content/departments/product-engineering/engineering/career-development/ic5/index.md

```diff
@@ -0,0 +1,29 @@
+# IC5 roles at Sourcegraph
+
+This page is a list of profiles for IC5 roles at Sourcegraph. The IC5 level is defined through these profiles, acknowledging that impact at IC5 and beyond varies between individuals. As they form the level definition, these profiles can be used in career growth conversations between ICs and EMs, in [calibration](../talent-review-process.md#calibration-meeting) discussions between EMs and directors.
+
+We recommend <a href="https://drive.google.com/drive/folders/1fnw5gmlLDDZb9wDodY3KfcTaM3OOQL7G" target="_blank" rel="noopener noreferrer">our company’s copy of “Staff Engineer” by Will Larson</a> as reading material.
+
+## Common traits of IC5 profiles
+
+While the realization of the IC5 roles in practice will vary, a common starting point is always a **business need** for the role: in some cases the business may need an IC to tackle broad, cross-cutting technical problems at the org or company level. In others, it may need a deep technical expert for a specific area, capable of devising solutions to highly complex and specialized technical problems.
+
+The business need may evolve over time, and IC5s are expected to adjust their focus accordingly: the expectation to proactively and continuously **identify and advocate for work that will bring strategic value** is another common trait of all IC5 positions.
```

Comment:
This is a great initial list. Some other potentially common traits (we don't have to include as part of this change but I wanted to share)
- Leading others: 
  - Ability to bring others along (convincing/influence) even in situations where there are strong opposing views on matters of organizational significance
  - Something around effective delegation but being overall accountable for the area they oversee?
- Time horizon that they're operating at. This should be higher than the expectations we have of our IC4s.